pages/blog/_posts/2020-08-20-building-software-for-a-telecom-core-network.md

```diff
@@ -0,0 +1,157 @@
+
+## The product
+
+One of the most common complaints people have with their Voicemail service in Europe is that you have to call in to your Voicemail to hear your messages. In order to play the message you have to navigate through a slow voice menu using your dialpad. We believed this would be easy to improve.
+
+Our product idea was simple. When Alice leaves a voicemail message for Bob, Bob receives this message either as an audio file or as a speech-to-text transcript.
+
+<div class="post-images">
+    <img src="/img/blog/building-software-for-a-telecom-core-network/voicebox-splash.png" alt="VoiceBox splash screen">
+    <img src="/img/blog/building-software-for-a-telecom-core-network/voicebox-home.png" alt="VoiceBox home screen">
+</div>
+
+The product idea isn’t what most people would call revolutionary, but almost all of the world's mobile operators have hermetically sealed core networks. This means it would be impossible to build this product without lawyering up and coming to some sort of agreement with one of them. The Working Group Two core network, however, is open and provides API access to Voicemail, MMS and SMS  (among other things), which is just what we need to build our product.
+
+## Developing on the Working Group Two platform
+
+To create a product on the Working Group Two platform, the first thing you have to do is create a free developer account at <a href="https://developer.wgtwo.com" target="_blank">https://developer.wgtwo.com</a>. Once you have signed up, you have to create an Organization and add a Product to that organization.
+
+You can specify what permissions your product will require in the `Product > Scopes` tab:
+
+<div class="post-images">
+    <img src="/img/blog/building-software-for-a-telecom-core-network/developer-portal-scopes.png" alt="Developer Portal Scopes Screen">
+</div>
+
+VoiceBox will need to: know the subscriber’s phone number, access their voicemail events, and access their voicemail messages (the audio files themselves).
+
+In an ideal world, VoiceBox would work like this:
+1. The subscriber signs in and enables the desired functionality in VoiceBox
+2. The next time the subscriber receives a voicemail, an event is fired by Working Group Two’s core network, which VoiceBox receives
+3. VoiceBox triggers an SMS/MMS send using our APIs
+4. The subscriber receives an SMS/MMS from the sender “VoiceBox”
+
+In a few months time this ideal world should be reality, but at the time of writing (mid August 2020) we’re missing the events API and the “Send from Product” API. Currently VoiceBox works like this (changes are highlighted):
+1. The subscriber signs in and enables the desired functionality in VoiceBox
+2. The next time the subscriber receives a voicemail, **VoiceBox will discover it by polling**
+3. VoiceBox triggers an MMS send using our APIs
+4. The subscriber receives an **MMS from their own number**
```

Comment:
The instances in these lists are full sentences; as far as I know they should be punctuated then.
(first article I found https://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points.html)

Reply:
Fixed (locally).